src/components/common/Dialogs/MintTokenDialog/MintTokenDialog.tsx

```diff
@@ -35,16 +34,10 @@ const validationSchema = object()
   .shape({
     forceAction: bool().defined(),
     annotation: string().max(4000).defined(),
-    mintAmount: string()
+    mintAmount: number()
```

Comment:
nitpick: It starts a number since the default value is `0`, but if you type anything in the value is saved as a string.

Reply:
Which is fine, isn't it? This just declares the validation expects a number (and we convert it using `toFinite` a few lines lower)
It was `number` in the Dapp, we changed it to string because we were storing a formatted value (e.g. `123,456`) instead of a raw one

Reply:
I would ask why the validation expects a number if we know it's going to be validating a string (albeit a string containing a number). I don't think it affects the actual validation at all (hence nitpick) but it is technically incorrect.

Reply:
I can't say it's incorrect - we want to validate a number, and therefore have access to `number` methods such as `moreThan`, `transform`.

Reply:
> we want to validate a number, and therefore have access to number methods

That's true, I was thinking of it as if it were an interface.